src/formBuilder/PredefinedGallery.js

```diff
@@ -133,6 +134,7 @@ export default function PredefinedGallery({
   const classes = useStyles();
   const schemaData = (parse(schema): { [string]: any }) || {};
   const uiSchemaData = (parse(uischema): { [string]: any }) || {};
+  const galleryMods = objectExcluding(mods, ['deactivatedFormInputs']);
```

Comment:
Sorry, I'm a bit confused -- I thought we were going to allow `deactivatedFormInputs` into the `PredefinedGallery`. I don't see a reason why we should necessarily exclude it automatically -- if the developer wanted to deactivate form inptus in the FormBuilder component but not the PredefinedGallery input, they should be able to by passing in different `mods` objects themselves.

Reply:
You're absolutely right. I forgot to remove this.  I have removed it now :)

Reply:
I also removed the `objectExcluding` function and tests since it is no longer needed